udata_hydra/db/resource_exception.py

```diff
@@ -50,25 +52,20 @@ async def insert(cls, resource_id: str, table_indexes: dict[str, str] | None = {
             if not valid:
                 raise ValueError(error)
 
-            async with pool.acquire() as connection:
-                q = f"""
-                    INSERT INTO resources_exceptions (resource_id, table_indexes)
-                    VALUES ('{resource_id}', '{json.dumps(table_indexes)}')
-                    RETURNING *;
-                """
-                return await connection.fetchrow(q)
-
-        else:
-            async with pool.acquire() as connection:
-                q = f"""
-                    INSERT INTO resources_exceptions (resource_id)
-                    VALUES ('{resource_id}')
-                    RETURNING *;
-                """
-                return await connection.fetchrow(q)
+        async with pool.acquire() as connection:
+            q = """
+                INSERT INTO resources_exceptions (resource_id, table_indexes, comment)
+                VALUES ($1, $2, $3)
+                RETURNING *;
+            """
+            return await connection.fetchrow(
+                q, resource_id, json.dumps(table_indexes) if table_indexes else None, comment
+            )
 
     @classmethod
-    async def update(cls, resource_id: str, table_indexes: dict[str, str] | None = {}) -> Record:
+    async def update(
+        cls, resource_id: str, table_indexes: dict[str, str] | None = {}, comment: str | None = None
+    ) -> Record:
```

Comment:
Same as above, do we want to take the opportunity to remove the default mutable value?